Lib/uu.py

```diff
@@ -153,7 +153,10 @@ def decode(in_file, out_file=None, mode=None, quiet=False):
             except binascii.Error as v:
                 # Workaround for broken uuencoders by /Fredrik Lundh
                 nbytes = (((s[0]-32) & 63) * 4 + 5) // 3
-                data = binascii.a2b_uu(s[:nbytes])
+                try:
+                    data = binascii.a2b_uu(s[:nbytes])
+                except:
```

Comment:
It is not a good idea to convert all exceptions into Error. You would want to be more specific about which types of errors are converted (in particular, BaseExceptions which are not also subclasses of Exception should not be masked in this way).